src/main/java/uk/gov/hmcts/darts/common/entity/RetentionPolicyTypeEntity.java

```diff
@@ -28,11 +28,14 @@ public class RetentionPolicyTypeEntity extends CreatedModifiedBaseEntity {
     private Integer id;
 
     @Column(name = "fixed_policy_key")
```

Comment:
We should probably add the nullable = false to the NOT NULL columns to make it a bit easier to understand looking at our entity?

Reply:
not sure it matters, but have added in anyway